### PR TITLE
upgrade deps

### PR DESCRIPTION
**Pull request by connyduck**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -80,19 +80,25 @@ android {
...
         transitive = true
     }
-    implementation 'androidx.core:core:1.0.1'
+    implementation 'androidx.core:core:1.0.2'
     implementation 'androidx.appcompat:appcompat:1.0.2'
     implementation 'androidx.browser:browser:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.0.0'
     implementation 'androidx.legacy:legacy-support-v13:1.0.0'
-    implementation 'com.google.android.material:material:1.1.0-alpha05'
+    implementation 'com.google.android.material:material:1.1.0-alpha06'
```
Comment (connyduck) [inline app/build.gradle]: Can this become stable please 🙄

File: app/build.gradle [hunk 2]
```diff
@@ -140,8 +146,8 @@ dependencies {
     implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
     implementation 'io.reactivex.rxjava2:rxkotlin:2.3.0'
     implementation 'com.squareup.retrofit2:adapter-rxjava2:2.5.0'
-    implementation 'com.uber.autodispose:autodispose-android-archcomponents:1.2.0'
-    implementation 'com.uber.autodispose:autodispose-ktx:1.2.0'
+    implementation 'com.uber.autodispose:autodispose-android-archcomponents:1.3.0'
+    implementation 'com.uber.autodispose:autodispose:1.3.0'
```
Comment (connyduck) [inline app/build.gradle]: they removed the `-ktx` artifact and merged it with the default one

File: app/src/main/res/values-night/styles.xml [hunk 3]
```diff
@@ -9,8 +9,6 @@
         <item name="colorSecondary">@color/tusky_blue</item>
         <item name="colorOnSecondary">@color/white</item>
 
-        <item name="colorSurface">@color/color_primary_dark_dark</item>
-
```
Comment (connyduck) [inline app/src/main/res/values-night/styles.xml]: for some reason the upgraded material library does not like this anymore?¿?

File: app/build.gradle [hunk 4]
```diff
@@ -80,19 +80,25 @@ android {
     }
 }
 
+project.tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+
```
Comment (connyduck) [inline app/build.gradle]: required by the new `autodispose` version

Comment (charlag) [reply app/build.gradle]: Resolved by accident, doesn't jvmTaget 1.8 mean android 26?..

Comment (connyduck) [reply app/build.gradle]: I have no idea tbh. We already compile Java for jvm 8, so it should be fine for Kotlin as well.

**Pull request merged**